### DISCUSSION TITLE
GitHub Pages for Festival Website

### DISCUSSION THREAD

**Discussion by BaltacMihai** [community/community#63962] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

I'm currently using GitHub Pages to build a festival website. The website seeks to inform visitors about the event and point them in the direction of a third-party ticketing website where they can buy tickets.

I've read the [GitHub Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service), but I still want to be sure that this kind of website does not violate the Terms.

I appreciate your time.
Mihai

**Comment by queenofcorgis** [comment, marked as answer]
Hey @BaltacMihai  :wave:. Yes, you are permitted to use GitHub Pages for informational purposes as described above, as long as it isn’t used for sensitive transactions like sending passwords or credit card numbers.

**Comment by AliYa-chen** [comment]
Hello, the festival website is in compliance with GitHub Pages and the relevant [GitHub terms of service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service), but the website is not allowed to collect sensitive content such as account passwords and credit cards.
